.circleci/test.sh

```diff
@@ -69,6 +69,9 @@ case ${CIRCLE_JOB} in
     OPENCL)
         run_unit_tests check
         ;;
+    FEATURE_COMPILATION)
+        run_unit_tests check
```

Comment:
@sladwa I'm not sure we get a lot of extra coverage by actually running tests -- WDYT? If so we shouldn't call this "compilation". If not then we should just remove this line and add a comment saying we only test compilation succeeds for this test.

Reply:
Hey @jfix71 , I wanted this CI only for compilation. As suggested i will add a comment.